web/src/features/fireWeather/components/WxDataForm.tsx

```diff
@@ -2,24 +2,41 @@ import React, { useEffect, useState } from 'react'
 import { makeStyles } from '@material-ui/core/styles'
 import { useHistory, useLocation } from 'react-router-dom'
 
-import WxStationDropdown from 'features/stations/components/WxStationDropdown'
 import TimeOfInterestPicker from 'features/fireWeather/components/TimeOfInterestPicker'
 import GetWxDataButton from 'features/fireWeather/components/GetWxDataButton'
 import { stationCodeQueryKey, timeOfInterestQueryKey } from 'utils/url'
-import WeatherMap from 'features/fireWeather/components/maps/WeatherMap'
+import WxStationDropdown from 'features/fireWeather/components/WxStationDropdown'
 
 const useStyles = makeStyles({
+  form: {
+    display: 'flex',
+    alignItems: 'center',
+    flexWrap: 'wrap',
+
+    '& fieldset, label, input': {
+      color: 'white !important',
+      borderColor: 'white !important'
+    },
+    '& .MuiChip-root': {
+      background: '#f0f0f0'
+    }
+  },
   stationDropdown: {
-    marginBottom: 10
+    marginRight: 16
+  },
+  toi: {
```

Comment:
I'd prefer to see this renamed to `timeOfInterest` - it's one less acronym to have to remember